src/VSCodeExtension/package.json.v.template

```diff
@@ -126,7 +126,7 @@
         "mocha": "^5.2.0",
         "yeoman-test": "^1.7.0",
         "yeoman-assert": "^3.1.0",
-        "@types/node": "^9.6.5",
+        "@types/node": "9.6.57",
```

Comment:
What do you think about committing `package-lock.json` to Git? It automatically pins all package version and should prevent this from happening in the future for any package (in theory).

Reply:
That seems like it would make a lot of sense, yes. I'd leave it to @ricardo-espinoza to make that decision.

Reply:
If we do, though, we should start using [`npm ci`](https://docs.npmjs.com/cli/ci) instead of `npm install` as the later will automatically update `package-lock.json`.

Reply:
Thanks for fixing the issue, @rmshaffer !

Sounds good, I'll give it a try to the `npm ci` approach.